src/config/environment.tsx

```diff
@@ -0,0 +1,11 @@
+export default {
```

Comment:
Should this be in the internal-infra repo?

Reply:
Absolutely